modules/core/PhotoSystem.js

```diff
@@ -78,10 +79,50 @@ export class PhotoSystem extends AbstractSystem {
         // Setup event handlers..
         urlhash.on('hashchange', this._hashchange);
         gfx.scene.on('layerchange', this._layerchange);
+        document.addEventListener('keydown', this._keydown);
```

Comment:
Rather than just adding this event listener to `document` , you should probably instead use `context.keybinding()` similar to how [`MapSystem` binds the "toggle osm" key](https://github.com/facebook/Rapid/blob/main/modules/core/MapSystem.js#L279).  

That `keybinding` component contains some special checks to make sure that the user doesn't have an `input` field focused... I haven't run the code in this PR, but I wonder whether just typing these letters `M`,`S`, or `K` elsewhere in Rapid - like in the 'name' field or in a changeset comment - will also toggle the layers.  🤔